test/mpi/configure.ac

```diff
@@ -251,6 +253,47 @@ AC_ARG_ENABLE(collalgo-tests,
                               [Run extended collective algorithm tests])],,
               [enable_collalgo_tests=no])
 
+# DTPools test generation
+AC_ARG_WITH(dtpools-datatypes,
+    [AC_HELP_STRING([--with-dtpools-datatypes=typelist],
+        [Semicolon separated list of MPI datatypes to use for DTPools tests
```

Comment:
This is actually my fault. Datatypes are coma separated not semicolon separated.

Reply:
I'll fix it.